src/foam/nanos/auth/User.js

```diff
@@ -692,6 +692,12 @@ foam.CLASS({
       externalTransient: true,
       columnPermissionRequired: true
     },
+    {
+      class: 'Boolean',
+      name: 'skipEmailVerification',
```

Comment:
This feature is based on the flow - not the user. Don't think this is where we want to go...